inference-engine/src/transformations/src/transformations/common_optimizations/common_optimizations.cpp

```diff
@@ -163,6 +164,9 @@ bool ngraph::pass::CommonOptimizations::run_on_function(std::shared_ptr<ngraph::
     manager.register_pass<ngraph::pass::ConstantFolding>();
     // need to convert to Gather-1 until plugins do not support Gather-7
     manager.register_pass<ngraph::pass::ConvertGather7ToGather1>();
+    // to enhance operation maintenance some plugins can choose to
+    // work with a single general Gather op with batch_dims
+    manager.register_pass<ngraph::pass::ConvertGather1ToGather7, false>();
```

Comment:
There is a transformation `EliminateUnsqueezeGather` which is called above. Should it be updated to support Gather7 as well?

Reply:
We already took care of newer Gather here
https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/transformations/src/transformations/common_optimizations/eliminate_unsqueeze_gather.cpp#L22